tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -459,13 +459,13 @@
                                         <RowDefinition/>
                                         <RowDefinition/>
                                     </Grid.RowDefinitions>
-                                    <TextBlock Text="Error logs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>
+                                    <TextBlock x:Uid="SummaryPage_ErrorLogs" Grid.Row="0" Grid.Column="0" Margin="0, 5, 0, 0"/>
                                     <StackPanel Orientation="Horizontal" Spacing="5" Grid.Row="1" Grid.Column="0">
                                         <TextBlock Text="{x:Bind ViewModel.FailedTasks.Count}" Foreground="{ThemeResource TextFillColorSecondary}"/>
-                                        <TextBlock Text="installation errors" Foreground="{ThemeResource TextFillColorSecondary}"/>
+                                        <TextBlock x:Uid="SummaryPage_InstallationErrors" Foreground="{ThemeResource TextFillColorSecondary}"/>

```

Comment:
Should this be one string with a variable?

Reply:
No.  I don't think so.  Why?  This isn't a list of errors.  Just a string "Installation errors"